README.md

```diff
@@ -57,8 +59,11 @@ If you need a fully functional installation with minimal configuration, follow t
 *If you want certificate verification, place your certificate in 'arangopipe/arangopipe_storage/cert'*
 
 ## Getting Started
```

Comment:
I would move the quickstart before the introduction

Reply:
done, please check @joerg84